app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/recordaudio/RecordAudioViewModel.kt

```diff
@@ -267,26 +266,13 @@ class RecordAudioViewModel @Inject constructor(
         viewModelScope.launch {
             recordAudioMessagePlayer.stop()
             recordAudioMessagePlayer.close()
+            state = state.copy(buttonState = RecordAudioButtonState.ENCODING, audioState = AudioState.DEFAULT)
 
-            val resultFile = if (state.shouldApplyEffects) {
-                try {
-                    state.originalOutputFile?.toPath()?.deleteIfExists()
-                } catch (exception: IOException) {
-                    appLogger.e("[$tag] -> Couldn't delete original audio file before sending audio file with effects.")
-                }
-                state.effectsOutputFile!!.toUri()
-            } else {
-                try {
-                    state.effectsOutputFile?.toPath()?.deleteIfExists()
```

Comment:
where are we deleting the files now? 🤔

Reply:
I had one issue with that. When I started uploading I got null pointer exception. Now it's fixed